utils/hct/hcttest.cmd

```diff
@@ -311,6 +311,13 @@ if "%TEST_USE_LIT%"=="1" (
     cmake --build %HLSL_BLD_DIR% --config %BUILD_CONFIG% --target check-all
     set RES_CLANG=!ERRORLEVEL!
     set RES_DXILCONV=!RES_CLANG!
+    rem check exec.
+    if defined EXEC_ADAPTER (
+        py %HLSL_SRC_DIR%/utils/lit/lit.py -v --no-progress-bar --param build_mode=%BUILD_CONFIG% --param clang_site_config=%HLSL_BLD_DIR%/tools/clang/test/lit.site.cfg --param clang_taef_exec_site_config=%HLSL_BLD_DIR%/tools/clang/test/taef_exec/lit.site.cfg %EXEC_ADAPTER% %HLSL_SRC_DIR%/tools/clang/test/taef_exec
+      ) else (
+        cmake --build %HLSL_BLD_DIR% --config %BUILD_CONFIG% --target check-clang-taef-exec
```

Comment:
Is there a reason why check-clang-taef-exec is not included in check-all?

Reply:
github pipeline doesn't run exec test.
check-all is matching this behavior.

Reply:
GitHub pipeline is running the tests with `noexec` parameter:

[call utils\hct\hcttest.cmd -$(configuration) noexec](https://github.com/microsoft/DirectXShaderCompiler/blob/bdefcfda03e7986f5da95c64fb5f4326f7420576/azure-pipelines.yml#L35)

I think is would make sense to include `check-clang-taef-exec` into `check-all`.

Reply:
Yea, `check-all` should be everything, including the exec tests.

Reply:
Fixed.

Reply:
I think we should keep skip_taef_exec, but they default value should be False. It is useful to run exec and non-exec tests separately.

Reply:
Fixed.